doc/examples/bundle_extraction.py

```diff
@@ -41,7 +41,7 @@
 
 sft_target = load_trk(target_file, "same", bbox_valid_check=False)
 target = sft_target.streamlines
-target_header = create_tractogram_header(atlas_file,
+target_header = create_tractogram_header(target_file,
                                          *sft_atlas.space_attributes)
```

Comment:
change `*sft_atlas.space_attributes ` to `*sft_target.space_attributes `

Reply:
good catch! will do